.github/workflows/publish.yaml

```diff
@@ -212,20 +214,71 @@ jobs:
         local-registry: bazel-central-registry
         templates-dir: this/.bcr
 
-    - name: Create Pull Request
-      id: create-pull-request
-      uses: peter-evans/create-pull-request@v7
-      with:
-        token: ${{ secrets.publish_token }}
-        path: bazel-central-registry
-        commit-message: ${{ steps.create-final-entry.outputs.short-description }}
-        base: ${{ inputs.registry_branch }}
-        branch: ${{ steps.create-final-entry.outputs.module-names }}-${{ inputs.tag_name }}
-        push-to-fork: ${{ inputs.registry_fork }}
-        title: ${{ steps.create-final-entry.outputs.short-description }}
-        body: |
-          Release: https://github.com/${{ inputs.repository }}/releases/tag/${{ inputs.tag_name }}
-
-          _Automated by [Publish to BCR](https://github.com/bazel-contrib/publish-to-bcr)_
-        maintainer-can-modify: true
-        draft: ${{ inputs.draft }}
+    - name: Push to fork
+      working-directory: bazel-central-registry
+      run: |
+        set -o errexit -o nounset -o pipefail
+
+        git config user.name "github-actions[bot]"
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git remote add authed-fork https://x-access-token:${{ secrets.publish_token }}@github.com/${{ inputs.registry_fork }}.git
+
+        BRANCH="${{ steps.create-final-entry.outputs.module-names }}-${{ inputs.tag_name }}"
+        git checkout -b "${BRANCH}"
+        git add .
+        git commit -m "${{ steps.create-final-entry.outputs.short-description }}"
+        git push --force authed-fork "${BRANCH}"
+
+    - name: Open pull request
+      working-directory: bazel-central-registry
+      run: |
+        set -o errexit -o nounset -o pipefail
+
+        REGISTRY_FORK="${{ inputs.registry_fork }}"
+        FORK_OWNER="${REGISTRY_FORK%%/*}"
+        BRANCH="${{ steps.create-final-entry.outputs.module-names }}-${{ inputs.tag_name }}"
+        TITLE="${{ steps.create-final-entry.outputs.short-description }}"
+        DRAFT="${{ inputs.draft }}"
+        MAINTAINER_CAN_MODIFY=true
+        BODY=$(cat <<END
+        Release: https://github.com/${{ inputs.repository }}/releases/tag/${{ inputs.tag_name }}
+
+        _Automated by [Publish to BCR](https://github.com/bazel-contrib/publish-to-bcr)_
+        END
+        )
+
+        REQUEST_BODY=$(jq --null-input \
+          --arg title "${TITLE}" \
+          --arg body "${BODY}" \
+          --arg head "${FORK_OWNER}:${BRANCH}" \
+          --arg base main \
+          --argjson draft "${DRAFT}" \
+          --argjson maintainer_can_modify "${MAINTAINER_CAN_MODIFY}" \
+          '{title: $title, body: $body, head: $head, base: $base, maintainer_can_modify: $maintainer_can_modify, draft: $draft}')
+
+        # https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#create-a-pull-request
+        RESPONSE_BODY=$(mktemp)
+        RESPONSE_CODE=$(curl \
```

Comment:
I'm not using the `gh` cli here because authenticating with a token seemed to require more permissions out of the PAT than simply using it in the request here.

Reply:
Maybe add that as a comment for anyone who reads this and wants to change it back